### PR TITLE
Stock Management - Receive stock + Correct Entry

### PR DESCRIPTION
**Pull request by anvmn**
#644 
#654

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2023-02-16 23-17

File: client/src/assets/scss/_new.scss [hunk 1]
```diff
@@ -4837,4 +4836,48 @@ div.stock-management {
...
+      border: 2px solid $color-light-gray;
+      color: $color-text;
+      padding: 10px;
+    }
+  }
+
+  // .overlay .ui.modal.diagnosis-popup {
+  //   margin-left: -250px;
+  //   width: 500px;
+  // }
```
Comment (AronNovak) [inline client/src/assets/scss/_new.scss:4882]:
```suggestion
```

perhaps

File: client/src/elm/Pages/Utils.elm [hunk 2]
```diff
@@ -1185,3 +1185,28 @@ customSaveButton language active msg label =
...
         [ text <| translate language label ]
+
+
+customPopup : Language -> Bool -> TranslationId -> ( Html msg, Html msg, msg ) -> Html msg
+customPopup language showWarning actionLabel ( topMessage, bottomMessage, action ) =
+    div [ class "ui active modal diagnosis-popup" ]
+        [ div [ class "content" ] <|
+            [ div [ class "popup-heading-wrapper" ]
+                [ img [ src "assets/images/exclamation-red.png" ] []
+                , div [ class "popup-heading" ] [ text <| translate language Translate.Warning ++ "!" ]
```
Comment (AronNovak) [inline client/src/elm/Pages/Utils.elm:1196]: Shouldn't the exclamation mark be part of the `translate.Warning`?

**Comment by AronNovak** [comment]
@anvmn I restarted CircleCI and now it's green.

File: client/src/elm/Pages/Utils.elm (hunk 2 above)
Comment (anvmn) [reply client/src/elm/Pages/Utils.elm:1196]: Well,  we can make it whatever we like, but as I see it, we translate a word.
So, no punctuation marks need to be part of it.

**Review by AronNovak — APPROVED** [review]

File: client/src/assets/scss/_new.scss (hunk 1 above)
Comment (anvmn) [reply client/src/assets/scss/_new.scss:4882]: This is removed at follow up PR (#664), so no need to handle it here.

**Pull request merged**